tests/search/test_solr.py

```diff
@@ -32,11 +35,13 @@ def test_quotes_query(self, mock_parse):
     @mock.patch('crane.search.solr.Solr._get_data', spec_set=True)
     @mock.patch('crane.search.solr.Solr._parse')
     def test_workflow_filter_true(self, mock_parse, mock_get_data, mock_filter):
-        mock_parse.return_value = [SearchResult('rhel', 'Red Hat Enterprise Linux')]
+        mock_parse.return_value = [
```

Comment:
Here, line 43, and line 62 seem to be unnecessarily re-formatted. The changes don't seem to be related to the functionality you're trying to add.